lib/Segments/DynamicSegments/Filters/WooCommerceSubscription.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace MailPoet\Segments\DynamicSegments\Filters;
+
+use MailPoet\Entities\DynamicSegmentFilterEntity;
+use MailPoet\Entities\SubscriberEntity;
+use MailPoetVendor\Doctrine\DBAL\Query\QueryBuilder;
+use MailPoetVendor\Doctrine\ORM\EntityManager;
+
+class WooCommerceSubscription implements Filter {
```

Comment:
We discussed it on Slack. The thing is that integration tests infrastructure doesn't have support for WooCommerce and WooCommerce subscriptions and it is not easy to add because of [the serialization issues](https://mailpoet.atlassian.net/browse/MAILPOET-3274).